Dockerfile

```diff
@@ -1,7 +1,5 @@
 FROM alpine:3.5
 
-MAINTAINER ZappiStore DevOps <devops@zappistore.com>
```

Comment:
Addressed in https://github.com/Intellection/docker-syslog-ng/pull/2/commits/2864e11f970b5d3292d61a40073f93c752f6da0a